Backend/admin/src/App.js

```diff
@@ -31,17 +30,16 @@ function App() {
         </Header>
         <Content>
           <Page>
-            <Route exact path="/" component={() => <div>HOME PAGE</div>} />
-            <Route exact path="/login" component={Login} />
-            <Route exact path="/questionary" component={Questionary} />
             <Switch>
-              <Route exact path="/questions" component={Questions} />
-              <Route exact path="/questions/create" component={AddQuestion} />
-              <Route
-                exact
-                path="/questions/:id/edit"
-                component={EditQuestion}
-              />
+              <Route exact path="/" component={() => <div>HOME PAGE</div>} />
```

Comment:
i think we can also drop the home page ?
or maybe add ass home the answers report from @bnenu  ?

Reply:
True. Let's clean routes after we have all of them. 
Also I don't like the redirect to the questions page for the unkown route. Because is feeling like the routing is not working always sends you to the question part. when in fact the route is not registered or is unknown :)